.github/workflows/gateway-conformance.yml

```diff
@@ -0,0 +1,72 @@
+name: Gateway Conformance
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  gateway-conformance:
+    runs-on: ubuntu-latest
+    steps:
+      # 1. Start the Kubo gateway
+      - name: Download Kubo gateway
+        uses: ipfs/download-ipfs-distribution-action@v1
+      - name: Start Kubo gateway
+        uses: ipfs/start-ipfs-daemon-action@v1
+
+      # 2. Download the gateway-conformance fixtures
+      - name: Download gateway-conformance fixtures
+        # TODO: use a release tag once https://github.com/ipfs/gateway-conformance/pull/19 is merged
+        uses: ipfs/gateway-conformance/.github/actions/extract-fixtures@e320c2b1f576f69732d824c4d48a1c37bb0ad320
```

Comment:
@galargh should we wait for that, or ok to merge and fix later?

Reply:
It's ok to attend to it later. We can merge 💪

Reply:
And we're using the released version now - this cuts 50s off of runtime.